e-commerce/shop-frontend/lib/backend.ts

```diff
@@ -8,6 +8,9 @@ type ArrayItemType<T> = T extends ReadonlyArray<infer U> ? U : never
 type OazapftsReturn<T extends (...args: any) => any> = DataFieldType<Awaited<ReturnType<T>>>
 
 export const backendAddress = "http://localhost:8082";
```

Comment:
Why doesn't an environment variable work?  That's funny

Reply:
I suspect this is happening because backendAddress is initialized before node is running and process.env is not populated. To make it work, the openapi layer might need to redone like payment